www/network.js

```diff
@@ -26,7 +26,9 @@ var exec = require('cordova/exec'),
 // Link the onLine property with the Cordova-supplied network info.
 // This works because we clobber the navigator object with our own
 // object in bootstrap.js.
-if (typeof navigator != 'undefined') {
+// Browser platform do not need to define this property, because
+// it is already supported by modern browsers
+if (cordova.platforId !== 'browser' && typeof navigator != 'undefined') {
```

Comment:
I think it should be cordova.platformId here, just found this problem when testing latest code on browser.